script/openqa-clone-job

```diff
@@ -119,9 +119,19 @@ Cloning a job will not fail if an asset is missing.
 
 =item B<--clone-children>
 
-Clone child jobs as well. This is useful if you want to clone a full
+Clone child jobs as well.
+
+=item B<--clone-parallel-children>
```

Comment:
if there is no difference why keep both?

Reply:
or add usage description. anything above ~1~ instance is multimachine in my mind. So i am not sure when `--clone-children` is suitable.

Reply:
But there *is* a difference. Since I'm likely going to make this the default it shouldn't matter much, though.

> So i am not sure when --clone-children is suitable.

This is about all kinds of dependencies (so it is also about any kind of chained dependencies). The specialty of MM tests is that they use parallel dependencies.

Reply:
thanks for replies. it is still spinning in my head the idea of a better naming but lets not block PR for now just to discuss this.

Reply:
I now made it default so we don't have to think about a better name anymore.